src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleKey.java

```diff
@@ -64,6 +65,19 @@ public final String toString() {
     return getName() + "@" + (getVersion().isEmpty() ? "_" : getVersion().toString());
   }
 
+  public static ModuleKey fromString(String s) throws ParseException {
```

Comment:
is this still needed?

Reply:
Thanks for catching this, remove the unused function.